apps/web-mzima-client/src/app/shared/components/language/language.component.scss

```diff
@@ -21,9 +21,9 @@
       transform: rotate(180deg);
     }
 
-    .mat-select-arrow-wrapper {
-      display: none;
-    }
+    // .mat-select-arrow-wrapper {
+    //   display: none;
+    // }
 
```

Comment:
Same as above, to keep the code clean and easy to read, try to avoid having too much commented out code. Its ok while doing development, but when the fix is ready, always delete the commented out code 👍

Reply:
Got it! I'll remove the commented-out code as requested to keep the codebase clean and easy to read. Appreciate the guidance!

Reply:
@Angamanga, changes made as requested ([d1ac6f3](https://github.com/ushahidi/platform-client-mzima/pull/841/commits/d1ac6f35524862c46cb58f39edbcd33ece3388e9)) Please review. Thanks!